### ISSUE TITLE
Working on V2(Beta) of homebridge?

### ISSUE THREAD

**Issue by banboobee** [simont77/fakegato-history#132]
Hi developers,

I found this library doesn't work on V2 of homebridge.

Keys of hap.Perms object are renamed to PAIRED_READ/PAIRED_WRITE from READ/WRITE. These changes cause Homekit to reject the response from "GET /acceraries", since corresponding values are null, and all existing accessories become "not responding".

Unfortunately, I don't have any ideas to deal with this issue other than ad-hock approach. I edited fakegato-history.js to refer the keys like this,

```
Perms.PAIRED_READ || Perms.READ
Perms.PAIRED_WRITE || Perms.WRITE
```

With these changes, beta version of V2 homebridge begins to work.

So, I would like to ask,

Are there anyone facing similar issue?
Is this temporally issue on V2 developing, or libraries need to handle?
If handling is essential, are there better ways to deal?

**Comment by NorthernMan54** [comment]
This is noted, going to be reviewing this and other components for HB 2.0 readiness this month

**Comment by banboobee** [comment]
Thank you for your reply.

Yes. I hope potential show stoppers will be idetified strategically.

**Comment by bwp91** [comment]
Following the commit history of this repo, it looks like the fixes have already been made in `v0.6.5`.

https://github.com/simont77/fakegato-history/commits/master/

I'm just unsure if a newer version of the package has been published to npm with the very latest fix commit.

**Comment by simont77** [comment]
Yes, it is on npm https://www.npmjs.com/package/fakegato-history

**Comment by smart7324** [comment]
It seems that `Perms.READ` and `Perms.WRITE` are still used in `v0.6.5`:

https://github.com/simont77/fakegato-history/blob/902372deda8a634e18f355c41bce7be104265f53/fakegato-history.js#L82

Could you please fix it and use `Perms.PAIRED_READ` and `Perms.PAIRED_WRITE` instead?

In the current version it keeps getting my accessories unresponsive and causing `No response` errors in Homebridge v2.0.0-beta. It also prevents new bridges to be paired as `undefined` permissions will lead to an `accessory out of compliance` error in Apple Home.

**Comment by NebzHB** [comment]
Hi, i've fixed that

https://github.com/simont77/fakegato-history/commit/e5007b646be1a6ccdff3170e3224f9efcb49e3d5

**Comment by benwebbbenwebb** [comment]
Thanks @NebzHB 

Will you or @simont77 be able to publish the new v0.6.6 release to NPM please?

**Comment by simont77** [comment]
Done
